CPAN/buildme.sh

```diff
@@ -674,7 +708,27 @@ function build {
         IO::Interface)
             build_module IO-Interface-1.06
             ;;
-        
+       
+	IO::Socket::SSL)
+            buildIOSocketSSL=1
+	    build_module Test-NoWarnings-1.02 "" 0
+	    build_module Net-IDN-Encode-2.400
+
+            tar_wrapper zxvf Net-SSLeay-1.82.tar.gz
+            cd Net-SSLeay-1.82
+            patch -p0 < ../NetSSLeay-SunOS-NoPrompt.patch
```

Comment:
This should probably for SunOS only, right?

Reply:
Well, it actually depends. It is named SunOS since I changed it for compiling on SunOS. In general the patch just comments out a user request from STDIN that pauses the building process until there is user input.
The patch goes with the defaults. I did not find any other "non-patchy" way.